kong/workspace_service_test.go

```diff
@@ -0,0 +1,84 @@
+package kong
+
+import (
+	"testing"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWorkspaceService(T *testing.T) {
+	runWhenEnterprise(T)
+	assert := assert.New(T)
+
+	client, err := NewClient(nil, nil)
+	assert.Nil(err)
+	assert.NotNil(client)
+
+	workspace := &Workspace{
+		Name: String("teamA"),
+	}
+
+	createdWorkspace, err := client.Workspaces.Create(defaultCtx, workspace)
+	assert.Nil(err)
+	assert.NotNil(createdWorkspace)
+
+	workspace, err = client.Workspaces.Get(defaultCtx, createdWorkspace.ID)
+	assert.Nil(err)
+	assert.NotNil(workspace)
+
+	workspace.Comment = String("new comment")
+	workspace, err = client.Workspaces.Update(defaultCtx, workspace)
+	assert.Nil(err)
+	assert.NotNil(workspace)
+	assert.Equal("teamA", *workspace.Name)
```

Comment:
Let's add an assert her to confirm `.Comment` is actually populated.

Can you also please add something in the `.Config` and `.Meta` to confirm they are correctly populated.